geoopt/optim/rwolfe.py

```diff
@@ -186,12 +190,11 @@ def init_loss(self):
             state = self.state[point]
             # project gradient onto tangent space
             grad = manifold.egrad2rgrad(point, grad)
-            grad_norms.append(grad.norm().item())
+            grad_norm = manifold.norm(point, grad)
```

Comment:
Be careful, norm and inner reduce only manifold dimensions, not batch ones. Was it what you meant?

Reply:
Good point. I did not design this optimizer with batches in mind (e.g. line search doesn't really make sense in the stochastic gradient descent setting). I will alter the optimizer so that it works correctly with batches of tensors.